qt_dotgraph/src/qt_dotgraph/dot_to_qt.py

```diff
@@ -295,5 +304,26 @@ def dotcode_to_qt_items(self, dotcode, highlight_level, same_label_siblings=Fals
             self.addEdgeItem(edge, nodes, edges,
                              highlight_level=highlight_level,
                              same_label_siblings=same_label_siblings)
+        return edges
+
+    def dotcode_to_qt_items(self, dotcode, highlight_level, same_label_siblings=False):
```

Comment:
Please do not move code around since it makes the diff much more complicated to review.